frontend/base_index.mustache

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    {% if settings.DEBUG %}
+      <link rel="icon" href="http://localhost:5173/static/img/GATE.svg">
+    {% else %}
+      <link rel="icon" href="/static/img/GATE.svg">
+      <link rel="stylesheet" href="/{{{ main_css }}}" />
+    {% endif %}
+    <title>{{open_braces}}settings.APP_NAME{{close_braces}}</title>
```

Comment:
I know it was me who did the `{{open_braces}}` trick, but would it maybe be cleaner to make this file a lodash template instead of a Mustache one so we don't have to worry about the clash of delimiters?

Reply:
Sounds sensible to me